packages/frontend/src/PortalForm/index.tsx

```diff
@@ -133,10 +181,13 @@ const MappingBar = () => {
             hours: hr,
             minutes: min,
           })
+
+          setFrom(DEFAULT_ZONE)
```

Comment:
Let's remove this. This will slow down entering connections when you enter a zone, which needs to be as fast as possible.